manager/sql/manager_sql.go

```diff
@@ -14,158 +14,50 @@ import (
-compiled 	varchar(511) NOT NULL UNIQUE,
-template 	varchar(511) NOT NULL UNIQUE
-)`,
-		`CREATE TABLE IF NOT EXISTS ladon_action (
-id       varchar(64) NOT NULL PRIMARY KEY,
-has_regex   bool NOT NULL,
-compiled varchar(511) NOT NULL UNIQUE,
-template varchar(511) NOT NULL UNIQUE
-)`,
-		`CREATE TABLE IF NOT EXISTS ladon_resource (
-id       varchar(64) NOT NULL PRIMARY KEY,
-has_regex   bool NOT NULL,
-compiled varchar(511) NOT NULL UNIQUE,
-template varchar(511) NOT NULL UNIQUE
-)`,
-		`CREATE TABLE IF NOT EXISTS ladon_policy_subject_rel (
-policy varchar(255) NOT NULL,
-subject varchar(64) NOT NULL,
-PRIMARY KEY (policy, subject),
-FOREIGN KEY (policy) REFERENCES ladon_policy(id) ON DELETE CASCADE,
-FOREIGN KEY (subject) REFERENCES ladon_subject(id) ON DELETE CASCADE
-)`,
-		`CREATE TABLE IF NOT EXISTS ladon_policy_action_rel (
-policy varchar(255) NOT NULL,
-action varchar(64) NOT NULL,
-PRIMARY KEY (policy, action),
-FOREIGN KEY (policy) REFERENCES ladon_policy(id) ON DELETE CASCADE,
-FOREIGN KEY (action) REFERENCES ladon_action(id) ON DELETE CASCADE
-)`,
-		`CREATE TABLE IF NOT EXISTS ladon_policy_resource_rel (
-policy varchar(255) NOT NULL,
-resource varchar(64) NOT NULL,
-PRIMARY KEY (policy, resource),
-FOREIGN KEY (policy) REFERENCES ladon_policy(id) ON DELETE CASCADE,
-FOREIGN KEY (resource) REFERENCES ladon_resource(id) ON DELETE CASCADE
-)`,
-	},
-}
-
-var migrations = map[string]*migrate.MemoryMigrationSource{
-	"postgres": {
-		Migrations: []*migrate.Migration{
-			{Id: "1", Up: sqlUp["1"], Down: sqlDown["1"]},
-			{
-				Id: "2",
-				Up: sqlUp["2"],
-			},
-			{Id: "3",
-				Up: []string{
-					"CREATE INDEX ladon_subject_compiled_idx ON ladon_subject (compiled text_pattern_ops)",
-					"CREATE INDEX ladon_permission_compiled_idx ON ladon_action (compiled text_pattern_ops)",
-					"CREATE INDEX ladon_resource_compiled_idx ON ladon_resource (compiled text_pattern_ops)",
-				},
-				Down: []string{
-					"DROP INDEX ladon_subject_compiled_idx",
-					"DROP INDEX ladon_permission_compiled_idx",
-					"DROP INDEX ladon_resource_compiled_idx",
-				},
-			},
-		},
-	},
-	"mysql": {
-		Migrations: []*migrate.Migration{
-			{Id: "1", Up: sqlUp["1"], Down: sqlDown["1"]},
-			{
-				Id:   "2",
-				Up:   sqlUp["2"],
-				Down: []string{},
-			},
-			{
-				Id: "3",
-				Up: []string{
-					"CREATE FULLTEXT INDEX ladon_subject_compiled_idx ON ladon_subject (compiled)",
-					"CREATE FULLTEXT INDEX ladon_action_compiled_idx ON ladon_action (compiled)",
-					"CREATE FULLTEXT INDEX ladon_resource_compiled_idx ON ladon_resource (compiled)",
-				},
-				Down: []string{
-					"DROP INDEX ladon_subject_compiled_idx",
-					"DROP INDEX ladon_permission_compiled_idx",
-					"DROP INDEX ladon_resource_compiled_idx",
-				},
-			},
-		},
-	},
-}
-
 // SQLManager is a postgres implementation for Manager to store policies persistently.
 type SQLManager struct {
-	db     *sqlx.DB
-	schema []string
+	db       *sqlx.DB
+	schema   []string
```

Comment:
Doesn't seem like this is being used any more

Reply:
Agreed. Removing this might require changing the signature of the `NewSQLManager` function, currently `func NewSQLManager(db *sqlx.DB, schema []string) *SQLManager`. I could change that to `func NewSQLManager(db *sql.DB) *SQLManager` or continue accepting schema and ignore it until a major version release?